ops/create-cluster.sh

```diff
@@ -42,7 +42,7 @@ readonly max_retry_count=50
 retries=0
 
 while [[ -z ${cassandra_host} || -z ${DEIS_ROUTER_HOST_ROOT} ]]; do
-   cassandra_host=$(kubectl --namespace=cassandra get svc cassandra-cluster-cassan-ext -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
+   cassandra_host=$(kubectl --namespace=cassandra get svc cassandra-cluster-cassan-ext -o jsonpath='{.spec.clusterIP}')
```

Comment:
This IP address is only pingable within the cluster. Try to login to the pod via `kubectl exec ANYPODNAME -n spark -it bash`. Once you're logged in you should be able to access the IP.